tuxemon/states/persistance/load_menu.py

```diff
@@ -6,7 +6,6 @@
 
 logger = logging.getLogger(__name__)
 
-
 class LoadMenuState(SaveMenuState):
```

Comment:
please don't change formatting unless it is makes the code comply with PEP8 and other standards.  PEP8 states that there should be two lines between code blocks at the root level.  removing this will make the code non-compliant,

Reply:
fixed